### PR TITLE
Add sponsor link in committee profile

### LINKED ISSUE #4361: Add links from committee profile pages to candidate profile pages.
### Summary
Currently, we only show Leadership PAC sponsor name on about-committee section in committee profile page and don't have the link to candidate profile page.
ex: http://www.fec.gov/data/committee/C00764878/?cycle=2020&tab=about-committee
So this ticket will add the link to candidate profile page.

### Related issues
[https://github.com/fecgov/fec-cms/issues/4270](https://github.com/fecgov/fec-cms/issues/4270)
[https://github.com/fecgov/fec-cms/pull/4358](https://github.com/fecgov/fec-cms/pull/4358)

### Action items
- [ ] Add link on Leadership PAC sponsor name to corresponding candidate profile page.

Screen Shot 2021-02-02 at 2 56 00 PM

### PR DESCRIPTION
**Pull request by fec-jli**
## Summary (required)
Currently, we only show Leadership PAC sponsor name string on about-committee section in committee profile page and don't have the link to candidate profile page.
ex: https://www.fec.gov/data/committee/C00326439/?cycle=2022&tab=about-committee
before

after this PR, we change sponsor name to name card and link to candidate profile page.

after

- Resolves #4361

_Include a summary of proposed changes._
1) get sponsor_candidates list from get_committees function in view.py, send to front-end. 
2) modify about-committee.jinja, the sponsor_candidate info will be displayed by the way of name card.

## Impacted areas of the application
committee profile page

List general components of the application that this PR will affect:
data/view.py
about-committee.jinja

## How to test
1) checkout branch
2) point any api
3) pytest
./manage.py test
4) ./manage.py runserver
5) compare local and prod urls:
(1) Presidential candidate:
http://127.0.0.1:8000/data/committee/C00693713/?cycle=2020&tab=about-committee
https://www.fec.gov/data/committee/C00693713/?cycle=2020&tab=about-committee

(2) multi-house:(candidate_id=H6LA04138 and candidate_id=H6IN03229
http://127.0.0.1:8000/data/committee/C00326439/?cycle=2022&tab=about-committee
https://www.fec.gov/data/committee/C00326439/?cycle=2022&tab=about-committee

(3) senate:
http://127.0.0.1:8000/data/committee/C00764878/?cycle=2020&tab=about-committee
https://www.fec.gov/data/committee/C00764878/?cycle=2020&tab=about-committee

(4) multi-senate (but same candidate_id=S0PA00434)
http://127.0.0.1:8000/data/committee/C00455741/?cycle=2016&tab=about-committee
https://www.fec.gov/data/committee/C00455741/?cycle=2016&tab=about-committee

## Reviewer
one developer, one design developer

### CODE REVIEW

**Review by JonellaCulmer — APPROVED** [review]
All looks well to me, @fec-jli. Great work. Thank you.

File: fec/data/templates/partials/committee/about-committee.jinja [hunk 1]
```diff
@@ -108,33 +95,57 @@
...
-                <span class="entity__type">{{ committee.committee_type_full }} candidate</span>
-
-                  {% if candidates[0].office == 'S' %}
-                <span class="entity__type">{{ candidates[0].state|fmt_state_full }}</span>
-                  {% elif candidates[0].office == 'H' %}
-                <span class="entity__type">{{ candidates[0].state|fmt_state_full }} - {{ candidates[0].district }}</span>
+        <tr>
+          <td class="figure__label">Authorizing candidate:</td>
+          <td class="figure__value">
+            <div class="callout callout--primary">
```
Comment (patphongs) [inline fec/data/templates/partials/committee/about-committee.jinja:103]: If we are restricting the width of the candidate callout card, we should restrict it here too by adding `style="width: 100%; max-width: 300px"`

**Review by patphongs — CHANGES_REQUESTED** [review]
@fec-jli The functionality is working well, I just have one requested change before approving.

**Review by patphongs — APPROVED** [review]
Thanks for making the change, looks good to me

File: fec/data/templates/partials/committee/about-committee.jinja [hunk 2]
```diff
@@ -108,33 +95,57 @@
...
+                {% elif candidate.office == 'S' or candidate.office == 'H' %}
+                    <p class="callout__subtitle u-no-margin">Candidate for {{ constants.house_senate_types[candidate.office] }} |</p>
+                    <p class="callout__subtitle u-no-margin">{{ constants.states[candidate.state] }} | {{ candidate.party_full |capitalize}}</p>
+                {% endif %}
+                  </div>
+                </div>
+              </div>
+              {% endfor %}
+          </td>
+        </tr>
```
Comment (patphongs) [inline fec/data/templates/partials/committee/about-committee.jinja:148]: @fec-jli I just noticed one more thing, we don't want to show the "Committee sponsor" section if the `sponsor_candidate_ids` is null. Can you add a check for that on this entire <tr>: {% if sponsor_candidates %} {% endif %}

**Review by johnnyporkchops — APPROVED** [review]
Looks good, we should commit to moving inline styles to a  stylesheet in the next pass of the template code.

**Pull request merged**